src/mds/Server.cc

```diff
@@ -1125,6 +1127,9 @@ void Server::evict_cap_revoke_non_responders() {
 }
 
 void Server::handle_conf_change(const std::set<std::string>& changed) {
+  if (changed.count("mds_replay_unsafe_with_closed_session")) {
+    replay_unsafe_with_closed_session = g_conf().get_val<bool>("mds_replay_unsafe_with_closed_session");
+  }
```

Comment:
Also need to add the config key to `MDSRankDispatcher::get_tracked_conf_keys()`

Reply:
done

Reply:
> Also need to add the config key to `MDSRankDispatcher::get_tracked_conf_keys()`

done